packages/react-app/src/components/dashboard.tsx

```diff
@@ -41,7 +41,15 @@ export default function Dashboard() {
   const theme = useTheme();
   const [dashboardModel, setDashboardModel] = useState<DashboardModel>();
   useEffect(() => {
-    getDashboardInfo().then(setDashboardModel);
+    let isSubscribed = true;
```

Comment:
does that really remove the console message ?

Reply:
And thx for helping us with the console messages :D

Reply:
Yah because dashboard its loaded so its get unmounted but the request still running, after that when promises end it try managet setState or props inside the unmounted component and get the lack.

Reply:
Oh i see

Reply:
The ideal should be cancel the request too.

Something like use that - https://github.com/ShivamS136/advanced-promise